project/urls.py

```diff
@@ -7,4 +7,5 @@
     path('admin/', admin.site.urls),
     path('accounts/', include('allauth.urls')),
     path('', views.Home.as_view(), name='home'),
+    path('heartbeat', views.Heartbeat)
```

Comment:
```suggestion
    path('heartbeat/', views.heartbeat)
```
I _think_ having the trailing slash is better because it will keep you from matching other things, like `/heartbeatandmore`. Either way, all the doc examples have it.